app/assets/stylesheets/application.scss

```diff
@@ -34,4 +34,4 @@ $govuk-assets-path: "/";
 @import "providers/search_results";
 @import "dropzone/src/dropzone";
 @import "dropzone/src/basic";
-@import "providers/summary_list_action_width";
+@import "providers/summary_list_action_width"
```

Comment:
Can this import be removed? It doesn't look like it is being used.

Reply:
The class `summary_list_action_width_auto` seems to be referenced in three different views, where or not they are still required is another question